.vscode/launch.json

```diff
@@ -0,0 +1,51 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug dry run",
+      "type": "Ruby",
+      "request": "launch",
+      "program": "${workspaceRoot}/bin/dry-run.rb",
+      "cwd": "${workspaceRoot}",
+      "useBundler": true,
+      "args": [
+        "${input:package_manager}",
+        "${input:repository}"
+      ]
+    }
+  ],
+  "inputs": [
+    {
+      "type": "pickString",
+      "id": "package_manager",
+      "description": "What type of package manager do you want to debug?",
+      "options": [
+        "bundler",
+        "cargo",
+        "composer",
+        "dep",
+        "docker",
+        "elm",
+        "go_modules",
+        "gradle",
+        "hex",
+        "maven",
+        "npm_and_yarn",
+        "nuget",
+        "pip",
+        "submodules",
+        "terraform"
+      ],
+      "default": "go_modules"
+    },
+    {
+      "type": "promptString",
+      "id": "repository",
+      "description": "Enter the name of the repository to scan.",
+      "default": "zonedb/zonedb"
+    }
```

Comment:
This is awesome but was having some issues figuring out how to set env variable/pass in the github token used to fetch private git repos. This is currently loaded in dry run if it's set as `LOCAL_GITHUB_ACCESS_TOKEN`. Do you know of a good way to be able to set this and have it loaded when running this debug script? Was looking at loading an env file in the devcontainer setup but this seems to blow up if file isn't there (think we want to gitignore this?). Maybe also annoying if this requires us to rebuild every time we change?

https://code.visualstudio.com/docs/remote/containers-advanced#_option-2-use-an-env-file

Reply:
@feelepxyz I think this can be done by adding the environment variable in `launch.json` and wouldn't require rebuilds. I've added a fixup commit that adds the variable with an empty value. Please try adding your token there, it should work.